source/testing/index.md

```diff
@@ -5,39 +5,71 @@ This application would likely include models such as `user` and `post`. It would
 also include interactions such as _login_ and _create post_. Let's finally
 assume that you would like to have [automated tests] in place for your application.
 
-There are two different classifications of tests that you will need:
-**Acceptance** and **Unit**.
+There are three different classifications of tests that you will need:
+**Acceptance**, **Unit**, and **Integration**.
 
 ### Acceptance Tests
 
-Acceptance tests are used to test user interaction and application flow. With
-the example scenario above, some acceptance tests you might write are:
+Acceptance tests are used to test user interaction and application flow.  The tests interact
+with the application in the same ways that a user would, by doing things like filling out
+form fields and clicking buttons.  Acceptance tests ensure that the features within
+a project are basically functional, and are valuable in ensuring the core features of a
+project have not regressed, and that the project's goals are being met.
+
+In the example scenario above, some acceptance tests one might write are:
 
 * A user is able to log in via the login form.
 * A user is able to create a blog post.
+* Saving a post successfully shows the list of prior posts.
 * A visitor does not have access to the admin panel.
 
 ### Unit Tests
 
-Unit tests are used to test isolated chunks of functionality, or "units", without
-worrying about their dependencies. Some examples of unit tests for the scenario
-above might be:
+Unit tests are used to drive isolated chunks of functionality, or "units". They are
+often most effective when written within an iterative TDD cycle.
```

Comment:
I'm a fan of TDD as well, but I'm not sure it's our place to advocating for it here, and I don't think it adds to the reader's understanding of Ember. I'd suggest cutting it out.